easybuild/easyconfigs/q/Qt6/Qt6-6.5.2-GCCcore-12.3.0.eb

```
@@ -0,0 +1,108 @@
+easyblock = 'CMakeNinja'
+
+name = 'Qt6'
+version = '6.5.2'
+
+homepage = 'https://qt.io/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+# disabling use of -ftree-vectorize is required to avoid compilation failures on some systems (e.g. Intel Skylake X)
+toolchainopts = {'vectorize': False}
+
+source_urls = [
+    'https://download.qt.io/official_releases/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/new_archive/qt/%(version_major_minor)s/%(version)s/single/',
+]
+sources = ['qt-everywhere-src-%(version)s.tar.xz']
+patches = [
+    # they may not be needed if qmake is not used?
+    # QT_CFLAGS_ARCH_HASWELL in cmake/QtCompilerOptimization.cmake should be patched?
+    # 'Qt5-5.13.1_fix-avx2.patch',
+    # 'Qt5-5.13.1_fix-qmake-libdir.patch',
+    'Qt6-6.5.2_fix_OF-Gentoo.patch',
+    # 'Qt5-5.15.2_fix-qtwebengine-breakpad-glibc-2.34.patch',
+]
+checksums = [
+    {'qt-everywhere-src-6.5.2.tar.xz': 'cde57be663d0f875759797298bdc37a936d517c39f2013e4e6ece5e12edeed12'},
+    # {'Qt5-5.13.1_fix-avx2.patch': '6f46005f056bf9e6ff3e5d012a874d18ee03b33e685941f2979c970be91a9dbc'},
+    # {'Qt5-5.13.1_fix-qmake-libdir.patch': '511ca9c0599ceb1989f73d8ceea9199c041512d3a26ee8c5fd870ead2c10cb63'},
+    {'Qt6-6.5.2_fix_OF-Gentoo.patch': '908f7dae7685c76fe651e52391e46a2337a3cd32dee70a5da54b154da7d4917c'},
+    # {'Qt5-5.15.2_fix-qtwebengine-breakpad-glibc-2.34.patch':
+    #  '74590de2b9e32f2c88123cb096c4f5c3001b00710aad096a4d16444a8e9eb991'},
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '1.9.5'),
+    ('CMake', '3.26.3'),
+    ('Ninja', '1.11.1'),
+    # deps for QtWebEngine
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('gperf', '3.1'),
+    ('Python', '3.11.3'),
+    ('re2c', '3.1'),
+]
+
+dependencies = [
+    ('double-conversion', '3.3.0'),
+    ('GLib', '2.77.1'),
+    ('PCRE2', '10.42'),
+    ('libpng', '1.6.39'),
+    ('LibTIFF', '4.5.0'),
+    ('libwebp', '1.3.1'),
+    ('JasPer', '4.0.0'),
+    ('HarfBuzz', '5.3.1'),
+    ('SQLite', '3.42.0'),
+    ('graphite2', '1.3.14'),
+    ('assimp', '5.2.5'),  # for Qt 3D
+    ('FFmpeg', '6.0'),
+    ('X11', '20230603'),
+    ('fontconfig', '2.14.2'),
+    ('html5lib', '1.1'),  # WebEngine
```

```suggestion
    ('Python', '3.11.3'),
    ('Python-bundle-PyPI', '2023.06'),  # WebEngine
```

[`Python-bundle-PyPi` contains `html5lb`](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb#L398), so we should use that.